### PR TITLE
Added test for ProgramBanListServiceTest. Closed #62

### LINKED ISSUE #62: Add test for ProgramBanListServiceTest
Add unit tests for Save method from ProgramBanListService in ProgramBanListServiceTest class.

### PR DESCRIPTION
**Pull request by Andrey31ts**

### CODE REVIEW

**Review by TopAcademy2022 — APPROVED** [review]

**Review by xs887z — DISMISSED** [review]

File: Tests/user-monitoring-gui-tests/user-monitoring-gui-tests/ProgramBanListServiceTest.cs [hunk 1]
```diff
@@ -10,11 +10,44 @@ namespace user_monitoring_gui_tests
...
+            {
+                programBanList.AddProgram($"Test{i}");
+            }
+
+            programBanListService.Save(programBanList, DataStorageArea.FILE);
+
+            var result = File.ReadLines("ProgramBanList.txt");
+
+            Assert.True(result.Count() == numberTestEntries);
+
```
Comment (TopAcademy2022) [inline Tests/user-monitoring-gui-tests/user-monitoring-gui-tests/ProgramBanListServiceTest.cs:37]: Add removing file "ProgramBanList.txt".

**Review by TopAcademy2022 — CHANGES_REQUESTED** [review]

File: Tests/user-monitoring-gui-tests/user-monitoring-gui-tests/ProgramBanListServiceTest.cs [hunk 2]
```diff
@@ -10,11 +10,44 @@ namespace user_monitoring_gui_tests
...
+         * \ brief SaveTest method to check if the transferred data is saved 
+         */
+
+        [Fact]
+        public void SaveTest()
+        {
+            /*
+             * - file exist verification
+             */
+            FileInfo file = new FileInfo("ProgramBanList.txt");
```
Comment (TopAcademy2022) [inline Tests/user-monitoring-gui-tests/user-monitoring-gui-tests/ProgramBanListServiceTest.cs]: Remove or move to Fixture.

File: Tests/user-monitoring-gui-tests/user-monitoring-gui-tests/ProgramBanListServiceTest.cs [hunk 3]
```diff
@@ -10,11 +10,44 @@ namespace user_monitoring_gui_tests
...
+
+            if (file.Exists)
+            {
+                file.Delete();
```
Comment (TopAcademy2022) [inline Tests/user-monitoring-gui-tests/user-monitoring-gui-tests/ProgramBanListServiceTest.cs]: Remove, see to 31 line comment.

**Review by TopAcademy2022 — CHANGES_REQUESTED** [review]

File: Tests/user-monitoring-gui-tests/user-monitoring-gui-tests/ProgramBanListServiceTest.cs [hunk 4]
```diff
@@ -10,11 +10,44 @@ namespace user_monitoring_gui_tests
      */
     public class ProgramBanListServiceTest
     {
-        private static IServerRequest _serverReqwest;
+        private static IServerRequest _serverRequest;
+
+        ProgramBanListService programBanListService = new ProgramBanListService(_serverRequest);
```
Comment (London3149-train) [inline Tests/user-monitoring-gui-tests/user-monitoring-gui-tests/ProgramBanListServiceTest.cs]: Add private modifier.

File: Tests/user-monitoring-gui-tests/user-monitoring-gui-tests/ProgramBanListServiceTest.cs [hunk 5]
```diff
@@ -10,11 +10,44 @@ namespace user_monitoring_gui_tests
...
 
-        ProgramBanListService programBanListService = new ProgramBanListService( _serverReqwest );
         ProgramBanList programBanList = new ProgramBanList();
```
Comment (London3149-train) [inline Tests/user-monitoring-gui-tests/user-monitoring-gui-tests/ProgramBanListServiceTest.cs]: Add private modifier.

**Review by London3149-train — CHANGES_REQUESTED** [review]

File: Tests/user-monitoring-gui-tests/user-monitoring-gui-tests/ProgramBanListServiceTest.cs (hunk 4 above)
Comment (TopAcademy2022) [reply Tests/user-monitoring-gui-tests/user-monitoring-gui-tests/ProgramBanListServiceTest.cs]: modifier*

**Review by TopAcademy2022 — APPROVED** [review]

**Review by SugarSalt9 — APPROVED** [review]

**Review by London3149-train — APPROVED** [review]

**Review by Mi-stik31 — APPROVED** [review]

**Review by Mi-stik31 — APPROVED** [review]

**Review by xs887z — APPROVED** [review]

**Pull request merged**